tf/envs/main.tf

```
@@ -55,3 +55,22 @@ module "kubernetes_dns" {
 provider "local" {
   version = "~> 1.2"
 }
+
+resource "aws_db_subnet_group" "publisher__test_rds_article_store_postgresql" {
+  subnet_ids = module.kubernetes_vpc.subnets
+}
+
+resource "random_password" "publisher__test_rds_article_store_postgresql_password" {
+  length = 50
+}
+
+resource "aws_db_instance" "publisher__test_rds_article_store_postgresql" {
+  allocated_storage = 20
+  engine = "postgres"
+  engine_version = "11.5"
+  instance_class = "db.t2.micro"
+  name = "articlestore"
+  username = "articlestore"
+  password = random_password.publisher__test_rds_article_store_postgresql_password.result
+  db_subnet_group_name = aws_db_subnet_group.publisher__test_rds_article_store_postgresql.name
```

We could do this:
```suggestion
  vpc_security_group_ids  = ['sg-018caeb91524d23ff']
```
This is `libero-eks--franklin-eks_worker_sg` which is what the worker nodes have. Untested, and should of course come out of the `kubernetes_vpc` module rather than being hardcoded.
But that is still wide, as the nodes don't just communicate with each other but also allow the master(s) and the ELBs to talk to them. We should instead create a new security group that allows traffic only from this one.

---

https://www.terraform.io/docs/providers/aws/r/security_group.html to create it.